docs/modules/deployment/pages/upgrade/rhel/up-rhel.adoc

```diff
@@ -57,17 +61,17 @@ yum-config-manager --disable opennms-repo-stable-*
 yum -y install java-11-openjdk java-11-openjdk-devel
 ----
 
-. Execute `runjava` to update which JVM {page-component-title} will use:
+. Execute `runjava` to set the JVM that {page-component-title} will use:
 +
 [source, console]
 ----
 ${OPENNMS_HOME}/bin/runjava -s
 ----
 
-. Check for configuration file changes, and update accordingly, using the files you backed up in xref:deployment:upgrade/diff.adoc#run_diff[Idenfity Changed Configuration Files].
+. Check for configuration file changes, and update accordingly using the files you backed up in xref:deployment:upgrade/diff.adoc#run_diff[idenfity changed configuration files].
```

Comment:
```suggestion
. Check for configuration file changes, and update accordingly using the files you backed up in xref:deployment:upgrade/diff.adoc#run_diff[identify changed configuration files].
```

Reply:
Good catch! I don't even know how I missed that. Thank you!

Reply:
I don't know how we missed before. There were lots of eyes on that section. :-)